src/tbbmalloc_proxy/proxy.cpp

```diff
@@ -434,11 +434,9 @@ const char* known_bytecodes[] = {
     "4883EC284885C974",       // release free()
     "4883EC284885C975",       // release _msize()
     "4885C974375348",         // release free() 8.0.50727.42, 10.0
-    "E907000000CCCC",         // release _aligned_msize(), _aligned_free() ucrtbase.dll
+    "E9********CCCC",         // release _aligned_msize(), _aligned_free() ucrtbase.dll and release _msize() ucrtbase.dll 10.0.14393.33
```

Comment:
Probably the explanation from the commit message deserves to be added in the comment.

Reply:
Done